api/src/main/java/org/pmiops/workbench/iam/IamServiceImpl.java

```diff
@@ -105,4 +123,30 @@ private void grantLifeScienceRunnerRole(String googleProject, List<String> petSe
                     .collect(Collectors.toList())));
     cloudResourceManagerService.setIamPolicy(googleProject, policy.setBindings(bindingList));
   }
+
+  /** Revokes life science runner role to list of service accounts. */
+  private void revokeLifeScienceRunnerRole(
+      String googleProject, List<String> petServiceAccountsLostAccess) {
+    com.google.api.services.cloudresourcemanager.model.Policy policy =
+        cloudResourceManagerService.getIamPolicy(googleProject);
+    List<com.google.api.services.cloudresourcemanager.model.Binding> bindingList =
+        Optional.ofNullable(policy.getBindings()).orElse(new ArrayList<>());
+    com.google.api.services.cloudresourcemanager.model.Binding binding =
+        bindingList.stream()
+            .filter(b -> b.getRole().equals(LIFESCIENCE_RUNNER_ROLE))
```

Comment:
nit: I usually recommend CONST.equals(variable) in general for better null safety